chore: using in-memory knex database in unit tests

RinkiyaKeDad opened the pull request:
<!--The content below is a suggested layout for writing your pull request description. You may extend or remove parts you deem relevant or irrelevant to your changes.-->
Fix for issue #2184 

<!--Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to [link to that issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).-->

I found two occurrences where SQLite database file was being created and have fix them both. Please let me know if some changes need to be made. Thanks.

## Types of changes

What types of changes does your code introduce to Graphback?

<!--Put an `x` in the boxes that apply-->

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please specify)

## Checklist

<!--Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.-->

- [x] I have read the [CONTRIBUTING](https://github.com/aerogear/graphback/blob/master/CONTRIBUTING.md) document.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

craicoverflow reviewed (changes requested):
I can see that tests are failing

RinkiyaKeDad commented:
Hey @craicoverflow! I was looking into those only and seems like I implemented it the wrong way. I've tried to do it again for ```DefaultCRUDService.test.ts ``` and have a doubt there which I've added as a comment. Can you please review that and tell me if I'm in the right direction. Once that it done I'll make similar changes to the other file also. Much thanks.

craicoverflow commented:
I actually think now that this will not work at all - the problem is that `migrateDB` does not accept a database, just a config which it uses to create a connection. This creates two separate in-memory databases (one during migrations) and another in the tests but this one does not have the migration history.

We will close this, but thanks for your PR!
